jdaviz/core/validunits.py

```diff
@@ -16,18 +17,19 @@ def supported_sq_angle_units(as_strings=False):
     return units
 
 
-def locally_defined_flux_units():
+def spectral_and_photon_flux_density_units():
     """
-    This function returns a list of string representations of flux units. This
-    list represents flux units that the unit conversion plugin supports
-    conversion to and from if the input data unit is compatible with items in the
-    list (i.e is equivalent directly or with u.spectral_density(cube_wave)).
-
+    This function returns an alphabetically sorted list of string representations
+    of spectral and photon flux density units. This list represents flux units
+    that the unit conversion plugin supports conversion to and from if the input
+    data unit is compatible with items in the list (i.e is equivalent directly
+    or with u.spectral_density(cube_wave)).
     """
     flux_units = ['Jy', 'mJy', 'uJy', 'MJy', 'W / (Hz m2)', 'eV / (Hz s m2)',
                   'erg / (Hz s cm2)', 'erg / (Angstrom s cm2)',
                   'ph / (Angstrom s cm2)', 'ph / (Hz s cm2)']
-    return flux_units
+
+    return sorted(flux_units)
```

Comment:
Now that I look at this again, this is basically a static list. Should this just be an app level (or package level) hardcoded attribute, rather than a function that always return the same thing?

Reply:
I see a thumbs up from Kyle. Does that mean we are getting rid of this function?

Reply:
i can move it to custom_units.py as a global variable like PIX2, does that work? i can make a note at the top of the file that it's for custom units and lists of units

Reply:
Fine by me if Kyle ok with that. Thanks!

Reply:
either way is fine with me - if it is essential to be sorted, then we can still have the sort done when assigning it to a variable and at least that will only be called once instead of repeatedly

Reply:
done